tests/ui/stability-attribute/check-stability-issue-138319-1.rs

```diff
@@ -0,0 +1,15 @@
+//@ run-pass
```

Comment:
```suggestion
//@ check-pass
```
For the other test too.

Reply:
Both tests can actually be merged together, I think.